src/pages/docs/tutorials/web-midi.md

```diff
@@ -0,0 +1,349 @@
+# Web MIDI
+
+This tutorial explains the basics of controlling an Elementary Audio app with the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API). The code discussed in this tutorial is available in the MIT-licensed [elementary-webmidi-demo](https://github.com/bgins/elementary-webmidi-demo) repository.
```

Comment:
Do you think we could frame this intro as explaining the basics of integrating Elem and MIDI in the generic sense, and for the sake of demonstrating we're specifically using WebMIDI in this case? Because one thing that I think is really helpful below is that you show how to think about reacting to midi events to update the elem graph, and that way of thinking applies regardless of how the midi events actually got to you (webmidi or in a plugin or over the network, etc etc)

Reply:
Rewrote this to reframe it. 👍 

I was thinking about whether we should update the title and file name (and thus slug) for the article to MIDI, but I worry that might be false advertisement. Probably best to keep Web MIDI in those parts. Thoughts?

Reply:
Caught up in Discord, agreed to keep it as is!